web/web/models/meter.py

```diff
@@ -176,8 +176,12 @@ def get_interval_coverage(self, obj):
         return MeterInterval.get_interval_coverage(coverage)
 
     def get_user_id(self, obj):
-        if obj.service_location.address.user:
-            return obj.service_location.address.user
+        users = obj.service_location.address.users
+        if users:
+            user_ids = []
+            for user in users:
+                user_ids.append(vars(user)["id"])
```

Comment:
instead of using `vars(user)["id]`, you can simply just do `user.id `- sqlalchemy makes it super easy to reference columns, etc!

Reply:
thanks, this is great